src/lib.rs

```diff
@@ -941,6 +943,53 @@ pub mod invariant {
         fn get_fee_tiers(&self) -> Vec<FeeTier> {
             self.fee_tiers.get_all()
         }
+
+        #[ink(message)]
+        fn get_position_ticks(&self, pool_key: PoolKey, offset: u16) -> Vec<PositionTick> {
```

Comment:
It looks like a liquidity tick interface. Position ticks should return pairs of (lower, upper) ticks for the user's position, skipping the offset amount of the user's position